spec/features/user_can_update_qty_of_item_in_cart_spec.rb

```diff
@@ -7,7 +7,10 @@
     item_1 = db_repo.items[0][0]
 
     visit '/items'
-    first(:button, 'Add to cart').click
+
+    within "div##{item_1.id}_add_button" do
+      first(:button, 'Add to cart').click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.